leptonai/api/workspace.py

```diff
@@ -182,6 +182,10 @@ def login(
     This function is intended to be used inside lepton deployments to log in to the
     workspace programmatically.
     """
+    if workspace_id is None and auth_token is None and url is None:
+        if WorkspaceInfoLocalRecord.get_current_workspace_id() is not None:
```

Comment:
actually we probably wanna check if current logged in id is the same as `LEPTON_WORKSPACE_ID`

Reply:
Yeah I thought about this for a while, but usually this shouldn't happen because it would be extremely strange if (1) the user hasn't specified anything, and (2) the env variable is set to LEPTON_WORKSPACE_ID and yet (3) some other workspace got logged in by the runtime... So I skipped it lol.